packages/fiori/src/IllustratedMessage.js

```diff
@@ -212,12 +212,33 @@ const metadata = {
 			type: IllustrationMessageSize,
 			defaultValue: IllustrationMessageSize.Auto,
 		},
+		/**
+		* Illustration breakpoint variant for the <code>Dialog</code> size.
+		*
+		* @private
+		* @type {String}
+		* @since 1.8.0
```

Comment:
Just change the since tag to 1.9.0 as the next version to include the newest development - 1.8.0 is already released.